commands/user_e2e_test.go

```diff
@@ -136,3 +136,41 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestListChannelsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("List channels", func(c client.Client) {
+		printer.Clean()
+
+		err := listChannelsCmdF(c, &cobra.Command{}, []string{s.th.BasicTeam.Name})
+		s.Require().Nil(err)
+
+		validChannelNames := append(
```

Comment:
I am not sure about the detail level of this test.

Depending on the client, the number of returned channels is different:
  6 for the normal client
  7 for local and sysadmin

To not test the details as requested in the ticket description I went for building this list of valid names.

The assertions then check that "all lines returned are channels and their names are in the valid set".

Reply:
Hey @Oppodelldog - I think this can be easily done if you use the `s.RunForSystemAdminAndLocal` for one case, and use `s.Run` for the other.